src/x86/sse2.rs

```diff
@@ -3588,6 +3661,102 @@ mod tests {
     }
 
     #[simd_test = "sse2"]
+    unsafe fn _mm_load_pd() {
+        let vals = &[1.0f64, 2.0, 3.0, 4.0];
+        let mut d = vals.as_ptr();
+
+        // Align d to 16-byte boundary
+        let mut offset = 0;
+        if (d as usize) & 0xf != 0 {
+            offset = (16 - (d as usize) & 0xf) >> 2;
```

Comment:
This is overflowing on travis, I wonder if the precedence here is making `(a - b) & 0xf` ?